content/guides/concepts/pinning.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Pinning"
+menu:
+    guides:
+        parent: concepts
+---
+
+## Summary
+
+IPFS nodes treat the data they store like a cache, meaning that there is no guarantee that the data will continue to be stored.  Pinning a CID tells an IPFS server that the data is important and mustn't be thrown away.
+
+You should pin any content you consider important, to ensure that content is retained long-term. Since data important to someone else may not be important to you, pinning lets you have control over the disk space used by your IPFS nodes.
+
+## Context 
+
+An IPFS node can store data based on different kinds of user events.  For instance, a person may add a file with `ipfs add ...`.  Or a piece of unknown IPFS content can be requested, and the node can consult with its peers to find this data - the result will be stored in the local cache.  These data can use different policies for managing the cache - `ipfs add ...` will automatically pin the content.  Other IPFS commands do not include automatic pinning.
```

Comment:
> For instance, a person may add a file with `ipfs add ...`.

We started this doc as something addressed to the reader in the second person as “you.” I think it would be good to carry that through here (we aren’t talking about how people throughout the world are using IPFS; we talking about _you_ managing _your_ node), so it would be better to say “for instance, you may add…”

> Or a piece of unknown IPFS content can be requested

I think it would be good to provide an example of how this happens that feels more concrete. Maybe something like:

“Or your node can store data you request it from other peers, such as when you load a web page through the gateway at http://localhost:8080/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco or run a command like `ipfs cat ...`.”

> These data can use different policies for managing the cache

I’m not sure this is the best way to frame it — there’s no policy for different kinds of data; only for different ways of adding the data to your node. I think you could just cut this part of the sentence and it would be a little clearer and more accurate.